internal/policy/policy_test.go

```diff
@@ -57,8 +57,10 @@ func TestInitializeNamespace(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		err = InitializeNamespace(repo)
-		assert.ErrorIs(t, err, ErrPolicyExists)
+		// Check if policy with zero hash is treated as uninitialized
+		if err := InitializeNamespace(repo); err != nil {
+			t.Fatal(err)
```

Comment:
You could just replace the assert with `assert.Nil(t, err)`. Also, let's use `Fatal` for test setup operations that fail rather than checks where we want to assert against an expected value.